package.json

```diff
@@ -5,7 +5,7 @@
     "access": "public"
   },
   "description": "GitHub webhook events toolset for Node.js",
-  "main": "src/index.js",
+  "main": "src/index.ts",
```

Comment:
you can remove the "main" key, `pika` sets it in `pkg/package.json` automatcially